build/DownloadSonarPlugin/DownloadSonarPlugin.csproj

```diff
@@ -1,16 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
     
     <!-- Not a product project or test project - completely exclude it from analysis/code coverage -->
     <SonarQubeExclude>true</SonarQubeExclude>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="14.3.0" />
-    <PackageReference Include="SharpCompress" Version="0.23.0" />
-    <PackageReference Include="SharpZipLib" Version="1.1.0" />
+    <PackageReference Include="SharpCompress" Version="0.32.1" />
+    <PackageReference Include="SharpZipLib" Version="1.3.3" />
```

Comment:
Mend/WhiteSource complained about both of these packages.
I'll manually re-run the scan once I've merged the changes to `branch-mend` to see how it handles them having been updated.